python/lsst/atmospec/utils.py

```diff
@@ -554,6 +575,27 @@ def getLinearStagePosition(exp):
     return linearStagePosition
 
 
+def getFilterAndDisperserFromFilterName(filterName):
+    """Get the filter and disperser from the filter name.
+
+    Parameters
+    ----------
+    filterName : `str`
+        The filter name.
+
+    Returns
+    -------
+    filter, disperser : `tuple` of `str`
+        The filter and the disperser names.
+    """
+    out = parseFilterName(filterName)
```

Comment:
I don't love `out` as a variable name here (it's fine internal to the parsing function though, of course, as that's the return value there) but that's not what you're returning here, it's what you're catching and using, so maybe `wheelConfig` or something?